templates/insurance-landing-page/insurance-landing-page.css

```
@@ -6,30 +6,31 @@
 .insurance-landing-page main .hero > div > div:not(:first-child) {
   align-items: start;
   flex-direction: column;
-  margin: 2rem 0px;
+  margin: 3rem 0px;
   max-width: 660px;
 }
 
-.insurance-landing-page main .hero > div > div h1{
+.insurance-landing-page main .hero > div > div h1 {
   margin-bottom: 0px;
 }
 
-.insurance-landing-page main .hero > div > div p{
-  color: #FFF;
+.insurance-landing-page main .hero > div > div p {
+  color: #fff;
```

You might as well use the existing variable for inverted text color
```suggestion
  color: var(--text-color-inverted);
```